### PR TITLE
Manylinux1 wheels

### PR DESCRIPTION
**Pull request by amitsaha**
Use "make manylinux1", you need to have docker installed. The library is
not being imported correctly in Python 2.6 with this error:

```
+ /opt/python/cp26-cp26m/bin//python -c 'import librabbitmq'
Traceback (most recent call last):
  File "<string>", line 1, in <module>
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/librabbitmq/__init__.py",
line 7, in <module>
    from amqp.protocol import queue_declare_ok_t
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/amqp/__init__.py",
line 48, in <module>
    from .basic_message import Message      # noqa
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/amqp/basic_message.py",
line 29, in <module>
    from .serialization import GenericContent
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/amqp/serialization.py",
line 34, in <module>
    from .exceptions import FrameSyntaxError
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/amqp/exceptions.py",
line 21, in <module>
    from .five import python_2_unicode_compatible
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/amqp/five.py", line
14, in <module>
    import vine.five
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/vine/__init__.py",
line 6, in <module>
    from .abstract import Thenable
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/vine/abstract.py",
line 7, in <module>
    from .five import with_metaclass
  File
"/opt/python/cp26-cp26m/lib/python2.6/site-packages/vine/five.py", line
230
    def with_metaclass(Type, skip_attrs={'__dict__', '__weakref__'}):
                                                   ^
SyntaxError: invalid syntax
Makefile:67: recipe for target 'manylinux1' failed
make: *** [manylinux1] Error 1
```

### CODE REVIEW

**Comment by amitsaha** [comment]
I don't expect this to be merged as it is. But let me know what are your thoughts and if we can start publishing them to PyPi?

File: Makefile [hunk 1]
```diff
@@ -63,6 +63,8 @@ $(RABBIT_TARGET):
 
 dist: rabbitmq-c $(RABBIT_TARGET)
 
+manylinux1: dist
```
Comment (auvipy) [inline Makefile]: is docker mendatory

Comment (amitsaha) [inline Makefile]: Yes, please see: https://github.com/pypa/python-manylinux-demo

**Comment by amitsaha** [comment]
Thanks @auvipy - not sure you wanted to merge the wheels themselves here? How can we go ahead and start publishing `manylinux1` wheels to PyPi?

**Pull request merged**